core-cube/row_key.hpp

```diff
@@ -0,0 +1,141 @@
+#ifndef ROWENCODER
```

Comment:
modify husky-kylin/src/core-cube/kv/row_key_encoder.hpp instead

Reply:
Change this https://github.com/husky-team/husky-kylin/blob/master/src/core-cube/kv/row_key_encoder.cpp#L50
and https://github.com/husky-team/husky-kylin/blob/master/src/core-cube/kv/row_key_encoder.hpp#L63

Reply:
@WuZihao1 You can get the data type of the column (string) from `TblColRef`  here (https://github.com/husky-team/husky-kylin/blob/master/src/core-cube/kv/row_key_encoder.cpp#L49) by getting its `ColumnDesc`. After you get the type string, you can get the DataType(https://github.com/husky-team/husky-kylin/blob/master/src/core-metadata/metadata/datatype/data_type.hpp) instance by `DataType::get_type()`. Then you check the family and then the type and get the corresponding column length, and store the column lengths in `RowKeyEncoder`